app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    student = Student.find_by email: params[:session][:email].downcase
+
+    if student&.authenticate(params[:session][:password])
+      redirect_to root_url
+    else
+      invalid_warning
+    end
+  end
+
+  private
+
+  def invalid_warning
```

Comment:
có thế này thôi thì cần gì tách method

Reply:
em xóa phần log in rồi, Tân làm phần này